src/applications/facility-locator/tests/components/UrgentCareAlert.unit.spec.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import UrgentCareAlert from '../../containers/UrgentCareAlert';
+import { showDialogUrgCare } from '../../utils/helpers';
+import { CLINIC_URGENTCARE_SERVICE, LocationType } from '../../constants';
+
+describe('<UrgentCareAlert>', () => {
+  it('Should always render when used.', () => {
+    const wrapper = shallow(<UrgentCareAlert />);
+    expect(wrapper.type()).to.not.equal(null);
+    const divs = wrapper.find('div');
+    expect(divs.length).to.equal(2);
+    const dl = wrapper.find('dl');
+    expect(dl.length).to.equal(1);
+    const dt = wrapper.find('dt');
+    expect(dt.length).to.equal(1);
+    const dd = wrapper.find('dd');
+    expect(dd.length).to.equal(1);
+    const button = wrapper.find('a');
+    expect(button.length).to.equal(1);
+    wrapper.unmount();
+  });
```

Comment:
It's up to you about making the test this thorough. A smoke test - component renders without crashing - might be enough imo